md/release-notes.md

```diff
@@ -16,6 +16,16 @@ modifying the Bonita installation.
 
 ## Feature removals
 
+## API behavior change
+
+* [addProcessComment()](https://documentation.bonitasoft.com/javadoc/api/7.7/org/bonitasoft/engine/api/ProcessRuntimeAPI.html#addProcessComment-long-java.lang.String-) method in ProcessRuntimeAPI has had a behavior change that went unnoticed in 7.3:
```

Comment:
What changed is the fix of this bug: https://bonitasoft.atlassian.net/browse/BS-14276

Operations on human tasks are asynchronous (as it should have been from 7.0.0). Hence all methods relying of Session to get the userID will find -1 as a value.

All scripts that want to perform an action on behalf of the user executing the task, should rely on the task assignee to do so. (There is no user logged hence the -1 value in the sessions; as the execution is asynchronous).

With the new API method, the script can leave a comment on behalf of the user. Note: this use case has never been considered before. As a comment was thought to be left by the user himself / herself.